docker-compose.yml

```diff
@@ -1,54 +1,71 @@
 version: '3'
 services:
   redis:
+    deploy:
+      replicas: 1
+      placement:
+        constraints:
+        - node.labels.role == opencti-core
+      restart_policy:
+        condition: on-failure
+        delay: 3s
+        max_attempts: 5
+        window: 60s
     image: redis:7.2.4
-    restart: always
     volumes:
       - redisdata:/data
-  elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:8.13.2
-    volumes:
-      - esdata:/usr/share/elasticsearch/data
-    environment:
-      # Comment-out the line below for a cluster of multiple nodes
-      - discovery.type=single-node
-      # Uncomment the line below below for a cluster of multiple nodes
-      # - cluster.name=docker-cluster
-      - xpack.ml.enabled=false
-      - xpack.security.enabled=false
-      - thread_pool.search.queue_size=5000
-      - logger.org.elasticsearch.discovery="ERROR"
-      - "ES_JAVA_OPTS=-Xms${ELASTIC_MEMORY_SIZE} -Xmx${ELASTIC_MEMORY_SIZE}"
-    restart: always
-    ulimits:
-      memlock:
-        soft: -1
-        hard: -1
-      nofile:
-        soft: 65536
-        hard: 65536
-  minio:
-    image: minio/minio:RELEASE.2024-01-16T16-07-38Z
-    volumes:
-      - s3data:/data
+      
+
+  nginx:
+    image: codeforafrica/cfa-opencti:0.0.6
+    deploy:
+      replicas: 1
+      placement:
+        constraints:
+        - node.labels.role == manager
+      restart_policy:
+        condition: on-failure
+        delay: 3s
+        max_attempts: 5
+        window: 60s
     ports:
-      - "9000:9000"
-    environment:
-      MINIO_ROOT_USER: ${MINIO_ROOT_USER}
-      MINIO_ROOT_PASSWORD: ${MINIO_ROOT_PASSWORD}    
-    command: server /data
-    restart: always
+      - "80:80"
+    depends_on:
+      - opencti
+
+
   rabbitmq:
     image: rabbitmq:3.13-management
+    deploy:
+      replicas: 1
+      placement:
+        constraints:
+        - node.labels.role == opencti-core
+      restart_policy:
```

Comment:
LGTM, if we've determined we're going to make this across all the services, then could leverage YAML anchors and to inherit the restart policy, DRY. Check [this](https://medium.com/@anasanjaria/code-reuse-in-docker-compose-using-yaml-anchor-feature-6cb2ff1d0427)

Reply:
@thepsalmist we have PR #1  and #2 of which I will close #2 since #3 has all the changes in it. What do you think about #1? should we merge it?

Reply:
I'll unblock you with closing on #1

Reply:
#1  is merged, you can review #2 and probably close, looks like the changes are outdated.

Reply:
@VinneyJ ☝🏽